addons/pos_event/__manifest__.py

```diff
@@ -0,0 +1,33 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': "POS - Event",
+    'category': "Hidden",
```

Comment:
I think category `Technical` would be a better choice (https://github.com/odoo/odoo/blob/master/odoo/addons/base/data/ir_module_category_data.xml)